google-datacatalog-hive-connector/tests/google/datacatalog_connectors/hive/scrape/metadata_database_scraper_test.py

```diff
@@ -37,11 +37,56 @@ def setUpClass(cls):
         MetadataScraperTestCase.hive_metastore_db_name = 'metastore'
         MetadataScraperTestCase.hive_metastore_db_type = 'postgresql'
 
-    @patch(f'{__SCRAPE_PACKAGE}.metadata_database_scraper.sessionmaker')
+    @mock.patch(f'{__SCRAPE_PACKAGE}.metadata_database_scraper.sessionmaker')
     def test_scrape_databases_metadata_should_return_objects(
             self, sessionmaker):  # noqa
 
-        sessionmaker.return_value = MockedSession
+        mocked_session = mock.MagicMock()
+
+        # Return the same instance, since session object
+        # uses a builder pattern.
+        mocked_session.return_value = mocked_session
```

Comment:
Don't MagicMock do this automatically? Unless I'm missing something, this statement and 4 next ones are not necessary.

Reply:
It does not, because I need to return the same instance in the builder pattern.
MagicMock will return a new instance and the code will not work, I tried to this using only MagicMock.

Reply:
Also, before I used the `MockedSession` class to simulate this builder pattern behavior, I was able to remove it and simply it in this PR.

Reply:
Ok, thanks for verifying. LGTM.